vercel/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "compilerOptions": {
```

Comment:
Since there aren't any type errors in the API anymore (since there is no API 😆 ), what do you think about adding back the `"extends": "@tsconfig/svelte/tsconfig.json"`?